src/Accounts/Authentication/Authentication/Clients/AuthenticationClientFactory.cs

```diff
@@ -67,6 +68,11 @@ public abstract class AuthenticationClientFactory
                 builder = builder.WithRedirectUri(redirectUri);
             }
 
+            builder.WithLogging((level, message, pii) =>
+            {
+                TracingAdapter.Information(string.Format("[MSAL] {0}: {1}", level, message));
```

Comment:
I can't remember, is this written to the debbug log, or is there a separate method for Debug messages in the Tracing Adapter?